runtime/src/swam/runtime/trace/JULTracer.scala

```diff
@@ -55,6 +59,29 @@ class JULTracer(conf: TraceConfiguration) extends Tracer {
 
 }
 
+object JULTracer {
+  def apply(traceFolder: String,
```

Comment:
As a general pattern, for these constructor, I tend to return an `F`, so that errors and effects are properly caught (for instance when loading the configuration). You can have a look at the way I did it for the `Engine`.